release-team/role-handbooks/release-lead.md

```diff
@@ -90,19 +94,22 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
   <tr>
     <td>1</td>
     <td><p>- Start the release cycle
+<p>- Insure you are joined to the following Google Groups: [kubernetes-milestone-burndown](https://groups.google.com/forum/#!forum/kubernetes-milestone-burndown),[kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release), [kubernetes-sig-leads](https://groups.google.com/forum/#!forum/kubernetes-sig-leads), [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev).
```

Comment:
Not sure why markdown isnt getting properly linked, probably due to lack of space between , and [ ?

Reply:
OOooh it's because to make a table there's inline HTML.  Those links should be html not markdown. :\